src/site/content/en/blog/app-shortcuts/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Get things done quickly with app shortcuts
+subhead: App shortcuts give quick access to a handful of common actions that users need frequently.
+authors:
+  - beaufortfrancois
+  - jungkees
+date: 2020-05-27
+updated: 2020-05-27
+hero: hero.jpg
```

Comment:
Weirdly enough this looks OK as a hero but a little grainy as a thumbnail:

![image](https://user-images.githubusercontent.com/4713486/82085479-f60a6980-96a1-11ea-8433-c64da6f98f90.png)

Maybe we can use a less-optimized version of the image?

Reply:
I realized we don't need a thumbnail at all if using the same image. So I've removed it.